spec/spec_helper.rb

```diff
@@ -18,3 +18,13 @@
     mocks.verify_partial_doubles = true
   end
 end
+
+RSpec::Matchers.define :embiggen do |expected|
```

Comment:
The naming feels a bit off here. `expect(X).to embiggen(Y)` makes it sound like Y is the thing being embiggened by X.

Naming the matcher `expand_to` or `embiggen_to` would make the spec read in a more grammatically accurate way.